filters/imagefilter.go

```diff
@@ -53,6 +53,15 @@ func HandleImageResponse(ctx filters.FilterContext, f ImageFilter) {
 }
 
 func handleImageTransform(out *io.PipeWriter, in io.ReadCloser, f ImageFilter) error {
+	var err error
+
```

Comment:
I should have asked you before, Can we make them into one function?. I mean adding in.Close() as the first statement in the first deferred function.